sdk/spring/spring-cloud-azure-starter-active-directory-b2c/pom.xml

```diff
@@ -177,14 +177,6 @@
                   <fileset dir="${project.basedir}/javadocTemp"/>
                   <fileset dir="${project.basedir}/sourceTemp"/>
                 </delete>
-                <echo>Copying ${project.basedir}/../README.md to
-                  ${project.basedir}/javadocTemp/README.md
-                </echo>
-                <copy file="${project.basedir}/../README.md" tofile="${project.basedir}/javadocTemp/README.md" failonerror="false"/>
-                <echo>Copying ${project.basedir}/../README.md to
-                  ${project.basedir}/sourceTemp/README.md
-                </echo>
-                <copy file="${project.basedir}/../README.md" tofile="${project.basedir}/sourceTemp/README.md" failonerror="false"/>
```

Comment:
maybe we can remove the whole `maven-antrun-plugin` part and `<classesDirectory>${project.basedir}/xxxTemp</classesDirectory>`  section in the jar plugin.

Reply:
Updated.